src/constants/models.ts

```
@@ -190,25 +193,25 @@ export const FIXTURE_CONFIG: Record<ComponentType, FixtureConfig> = {
   // Shower: PROCEDURAL_FIXTURES.find(f => f.name === 'Shower')!
 };
 
-// Helper function to get wall buffer for an object
+// Helper function to get wall buffer for an object - ALL VALUES IN CENTIMETERS
 export const getObjectWallBuffer = (
   objectType: ComponentType,
   scale: number = 1.0
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)